frappe/public/js/frappe/ui/field_group.js

```diff
@@ -86,7 +86,7 @@ frappe.ui.FieldGroup = frappe.ui.form.Layout.extend({
 			var f = this.fields_dict[key];
 			if(f.get_value) {
 				var v = f.get_value();
-				if(f.df.reqd && is_null(v))
+				if(f.df.reqd && is_null(strip_html(cstr(v))))
```

Comment:
This will break for `Code` and `HTML Editor` fields, say I add "<div></div>" in an HTML Editor field, it is a valid value to have, however it will mark it as empty.

An alternative is to have this check exclusively for `Text Editor`